adream_kitchen:
  position:
    z: 1.75
  polygon_path: package://overworld/models/areas/adream_kitchen.obj
  half_height: 1.75
  hysteresis: 0.2

adream_bedroom:
  position:
    z: 1.75
  polygon_path: package://overworld/models/areas/adream_bedroom.obj
  half_height: 1.75
  hysteresis: 0.2

adream_saloon:
  position:
    z: 1.75
  polygon_path: package://overworld/models/areas/adream_saloon.obj
  half_height: 1.75
  hysteresis: 0.2

eve_area:
  position:
    z: 0.75
  half_height: 0.75
  radius: 0.5
  hysteresis: 0.05
  owner: base_footprint